docs/configuration.md

```diff
@@ -0,0 +1,457 @@
+## Configuration
+Configuration parameters could be provided to Playkit JS upon instantiation of the player instance.
```

Comment:
I'm not clear what you mean here? When a player instantiates (begins??), the configuration parameters can be provided? And again, why are we still using playkit? Can you clarify both?